bento_sdk/src/Form/BentoSettingsForm.php

```
@@ -0,0 +1,765 @@
+    }
+  }
+
+  /**
+   * Validates credential changes for additional security.
+   *
+   * @param array $form
+   *   The form array.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   */
+  private function validateCredentialChanges(array &$form, FormStateInterface $form_state): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$form'. (Unused Code Rules)

(UnusedFormalParameter)